packages/client/src/features/Leaderboard/Leaderboard.tsx

```diff
@@ -1,5 +1,51 @@
 import type { FC } from 'react';
+import { useIntl } from 'react-intl';
+
+import { Typography } from 'antd';
+import { CrownTwoTone } from '@ant-design/icons';
+import { messages, data } from './common';
 
 import './Leaderboard.scss';
 
-export const Leaderboard: FC = () => <div>Leaderboard</div>;
+const { Title } = Typography;
+
+type LeaderboardPayer = {
+  place: number;
+  nick: string;
+  profit: number;
+  gameTime: string;
+};
+
+export const Leaderboard: FC = () => {
+  const { formatMessage: fm } = useIntl();
+  const leaderboardList: Array<LeaderboardPayer> | [] = data ?? [];
+
+  return (
+    <div className='leaderboard'>
```

Comment:
Здесь, наверное, случайно получились одинарные кавычки

```suggestion
    <div className="leaderboard">
```